esp-backtrace/src/lib.rs

```diff
@@ -1,9 +1,9 @@
 #![allow(rustdoc::bare_urls, unused_macros)]
 #![cfg_attr(
-    any(nightly_before_2024_06_12, nightly_since_2024_06_12),
+    target_arch = "xtensa",
+    feature(asm_experimental_arch),
     feature(panic_info_message)
 )]
```

Comment:
As far as I can tell, this is just protecting against a single `nightly` version which is past us now. These changes were made to prevent a warning when building with newer `nightly` versions.